eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  ServiceDirectory: 'not-specified'
```

Comment:
I think I would make this a little more general and allow any path but for our client template pass in `sdk/${{ parameters.ServiceDirectory }}`

Reply:
I was expecting this to be renamed to simply Directory and the `sdk/${{ parameters.ServiceDirectory }}` be passed to the caller of template (I added another comment there).